tests/suites/accessibility.js

```diff
@@ -0,0 +1,29 @@
+﻿module('Accessibility', {
```

Comment:
Added this file to do unit tests for accessibility.  This has a single test for now to test the generated label for a calendar day.